pingora/examples/client.rs

```diff
@@ -12,29 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use anyhow::Error;
```

Comment:
Prefer using the `pingora_error::{Result, Error}` types here. Those boxed error types are used throughout the codebase and equally suitable to be used in these examples. It also avoids pulling in an extra dependency, even if it is a dev-dependency.

I'll give some examples of ways to update the code accordingly.

Reply:
Sure no problem :+1:  5833556e5f8d9ccb346ef586cb1b5f51add64357